rails/deprecated.md

```diff
@@ -0,0 +1,19 @@
+This image is officially deprecated in favor of [the standard `ruby` image](https://hub.docker.com/_/ruby/), and will receive no further updates. Please adjust your usage accordingly.
```

Comment:
Yeah, I was planning on doing so, but calling it out explicitly and giving it a specific EOL date probably makes sense. :+1: